### DISCUSSION TITLE
Enable colorblind theme for mobile

### DISCUSSION THREAD

**Discussion by stephenyeargin** [community/community#72293] [Mobile]
### Select Topic Area

Product Feedback

### Body

This is a rather minor request, but I'd like for the PR review and status check icons to align with the same colors as the web UI when the colorblind theme is enabled. Helps me see things correctly at a glance.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.0 (113710104)
Device: iPhone16,1 (iOS 17.0.3)
</details>

**Comment by mattruggio** [comment]
Hey @stephenyeargin, thanks for the request! We are always looking to make our web and mobile experiences more accessible. I am going to create an internal mobile feature request.

**Reply by peterkos** [reply]
Hello,

Might be good to expand to its own issue, but the mobile app is also missing colorblind friendly diff colors. I'm red/green colorblind, so doing reviews on the go is quite difficult as I cannot easily discern added and removed lines 😅
